slices/python3.12-minimal.yaml

```diff
@@ -11,6 +11,7 @@ slices:
       - libpython3.12-minimal_libs
       - zlib1g_libs
     contents:
+      /usr/bin/python3: {symlink: /usr/bin/python3.12}
```

Comment:
This probably needs a bit of discussion, should I perhaps not add this and create it manually in the tests? Until we support the slice precedence? @cjdcordeiro

Reply:
Yup, that would be better. We are not installing `python3` paths currently.